### PR TITLE
Conda Windows package

### PR DESCRIPTION
**Pull request by tschoonj**
WIP

### CODE REVIEW

**Comment by tschoonj** [comment]
@ericdill Off to a bad start it seems...

**Comment by SylvainCorlay** [comment]
The Python 2 windows failure due to https://github.com/conda/conda-build/issues/188

**Comment by tschoonj** [comment]
@ericdill Looks like it's finally working, even the cython generated bindings which I couldn't get working locally for some reason...

Need to some cleanup as it appears to be dragging my SWIG installation along as well. Is there a way for me to see which files end up in the package?

**Comment by tschoonj** [comment]
Looks ready to me. Any objections to merge-squashing this one in?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,33 +8,51 @@ source:
...
+  patches:
+    - conda-windows.patch  # [win]
+    - xrayglob_inline.c.patch  # [win]
+    - swig-fix.patch  # [win]
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
+  skip: False
```
Comment (ericdill) [inline recipe/meta.yaml]: Just delete this line.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,33 +8,51 @@ source:
...
 
 requirements:
   build:
     - python
```
Comment (ericdill) [inline recipe/meta.yaml]: add `toolchain` as a build dep

**Comment by tschoonj** [comment]
Ok changes pushed!

The `xraylib` command does not work on Anaconda, as it would not be able to find the help text files. In my xraylib SDK for Windows, I rely on the registry to find them as the installer writes entries in it, but not sure how that would work here.

**Comment by ericdill** [comment]
> The xraylib command does not work on Anaconda, as it would not be able to find the help text files. 
Ah ok cool. Can you add that as a comment in the recipe so that future us remembers why win is being skipped here?

Also add this to the test/commands section:
```yaml
    - conda inspect linkages -p $PREFIX wcslib  # [not win]
    - conda inspect objects -p $PREFIX wcslib  # [osx]
```

This provides a record about what libraries are being linked against and if they're provided by conda packages or the system.

**Comment by ericdill** [comment]
That's weird. I have never seen this error before on conda inspect linkages...

```
+ conda inspect linkages -p /feedstock_root/build_artefacts/xraylib_1491835136556/_t_env wcslib
Package wcslib is not installed in /feedstock_root/build_artefacts/xraylib_1491835136556/_t_env
```

@pkgw Have you seen this issue before?

**Comment by pkgw** [comment]
@ericdill The last argument to those inspect commands should be the name of your package; it looks like they weren't updated from the recipe you were using as a template. That's why I've started using `$PACKAGE` as the last argument; `conda build` sets that environment variable to be what you want, and the `inspect` commands aren't run on Windows so you don't need to worry about Windows environment variable syntax.

**Comment by ericdill** [comment]
@pkgw Oh dear. Thanks for finding that here. That's what I get for blindly copy-pasting myself 🤦‍♂️ 🤦‍♂️

**Comment by tschoonj** [comment]
Looks like I got it sorted out by using `$PKG_NAME` instead of `$PACKAGE`

**Pull request merged**